aws/logs_monitoring/template.yaml

```diff
@@ -724,7 +744,7 @@ Metadata:
           - ExcludeAtMatch
           - IncludeAtMatch
       - Label:
-          default: Experimental (Optional)
+          default: Fetch Lambda Tags (Optional)
```

Comment:
I think we can merge it into the existing `Advanced (Optional)` group below.

Reply:
✅